h2o-r/h2o-package/R/models.R

```diff
@@ -3927,20 +3928,34 @@ h2o.cross_validation_predictions <- function(object) {
 #'                         max_depth = 5,
 #'                         learn_rate = 0.1)
 #' h2o.partialPlot(object = prostate_gbm, data = prostate, cols = c("AGE", "RACE"))
+#'
+#' iris_hex <- as.h2o(iris)
+#' iris_gbm <- h2o.gbm(x = c(1:4), y = 5, training_frame = iris_hex)
+#'
+#' # one target class
+#' h2o.partialPlot(object = iris_gbm, data = iris_hex, cols="Petal.Length", targets=c("setosa"))
+#' # three target classes
+#' h2o.partialPlot(object = iris_gbm, data = iris_hex, cols="Petal.Length", targets=c("setosa", "virginica", "versicolor"))
 #' }
 #' @export
 
 h2o.partialPlot <- function(object, data, cols, destination_key, nbins=20, plot = TRUE, plot_stddev = TRUE,
-                            weight_column=-1, include_na=FALSE, user_splits=NULL, col_pairs_2dpdp=NULL, save_to=NULL,
-row_index=-1) {
+                            weight_column=-1, include_na=FALSE, user_splits=NULL, col_pairs_2dpdp=NULL, save_to=NULL, 
+                            row_index=-1, targets=NULL) {
```

Comment:
was the api validated in #h2o-api-updates?

Reply:
Nope, I will do.